### PR TITLE
FlowerBucket and FlowerPack added

### PR DESCRIPTION
**Pull request by frontfliip**
FlowerBucket and FlowerPack added

### CODE REVIEW

File: src/main/java/shop/FlowerBucket.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+public class FlowerBucket {
+    private final List<FlowerPack> packs = new ArrayList<>();
+    public void add(FlowerPack flowers){
+        this.packs.add(flowers);
+    }
+    public double getPrice(){
+        double price = 0;
+        for (int i = 0; i < packs.size(); i++) {
+            price += (packs.get(i).getFlower().getPrice() * packs.get(i).getQuantity());
+        }
```
Comment (nikbackflip) [inline src/main/java/shop/FlowerBucket.java]: Please move packet price calculation to FlowerPack

Comment (frontfliip) [reply src/main/java/shop/FlowerBucket.java]: fixed

Comment (nikbackflip) [reply src/main/java/shop/FlowerBucket.java]: Looks good.
Maybe replace the loot with stream

**Review by nikbackflip — APPROVED** [review]
Approved after comment addressed

**Pull request merged**